components/src/banner/banner.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="tw-py-2.5 tw-px-4 tw-text-contrast tw-flex tw-gap-2" [ngClass]="bannerClass">
+  <div class="tw-block">
+    <i class="bwi {{ icon }} tw-align-middle" *ngIf="icon" aria-hidden="true"></i>
```

Comment:
I don't think we should use interpolation in `class` like this, this is what `ngClass` is for.